spec/support/true_layer_mock_data.rb

```diff
@@ -7,75 +7,81 @@ module TrueLayerHelpers
         provider_id: "lloyds",
       },
     },
-    account_holders: [{
-      full_name: Faker::Name.name,
-      date_of_birth: Faker::Date.backward.to_time.utc.iso8601,
-      addresses: [{
-        address: Faker::Address.street_address,
-        city: Faker::Address.city,
+    account_holders: [
```

Comment:
These are alignment changes. Personally I find it much easier to follow now, but feel free to object if you find it more difficult 👍

Reply:
Fine by me :+1: